hardware/_Sidebar.md

```diff
@@ -2,5 +2,8 @@
 --------------------------------------------------------
 [Home](Home) > [Hardware](Hardware-Boot-Camp)
 
-* [Schematic-Design-Guide](Schematic-Design-Guide)
-* [Wireless-Products-Protocols-and-Applications](Wireless-Products-Protocols-and-Applications)
+* [Wireless Products Protocols and Applications](Wireless-Products-Protocols-and-Applications)
+* [Schematic Design Guide](Schematic-Design-Guide)
+* [PCB Layout Guide](PCB-Layout-Guide)
```

Comment:
Suggest "PCB Layout Design Guide" for consistency with "Schematic Design Guide"

Reply:
Corrected.